examples/data-objects/prosemirror/src/prosemirror.tsx

```diff
@@ -50,8 +50,8 @@ function insertMarkers(
 	beginMarkerProps[reservedRangeLabelsKey] = [treeRangeLabel];
 	beginMarkerProps[nodeTypeKey] = nodeType;
 
-	text.insertMarker(endMarkerPos, ReferenceType.Simple, endMarkerProps);
 	text.insertMarker(beginMarkerPos, ReferenceType.Simple, beginMarkerProps);
+	text.insertMarker(endMarkerPos, ReferenceType.Simple, endMarkerProps);
```

Comment:
this function is really weird, as it requires the caller to compute the endMarkerPos which must include the length of the marker that will be inserted at beginMarkerPos.  It looks like its only used in a single place. i'd just remove endMarkerPos. Then you can either insert the end first at beginMarkerPos, or change it to be second at beginMarkerPos+1